app/components/sy-topnav/template.hbs

```diff
@@ -61,7 +61,7 @@
       <li class="nav-top-list-item">
         <LinkTo @route="users.edit" @model={{this.session.data.user.id}}>
           <FaIcon @icon="user" />
-          {{this.session.data.user.fullName}}
```

Comment:
before we had fullName on the user and now thats gone? How come?

Reply:
because there is an issue with `ember-simple-auth` or `ember-simple-auth-oidc`,
because in the first time login, the `session.data` is return an instance of ember data model, but after refreshing the token, it's return just an object of the user information, so i can not use the helper that is defined in `User` data model